src/wp-admin/css/revisions.css

```diff
@@ -212,10 +212,6 @@ table.diff col.ltype {
 	width: 30px;
 }
 
-table.diff tr {
-	background-color: transparent;
-}
-
```

Comment:
This was removed because I couldn't see it was affecting anything, the table tr was already a transparent background. I'll revert though just in case and so that this can just focus on color updates rather than any other cleanup.